pkg/kubectl/cmd/log.go

```diff
@@ -93,6 +93,7 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"log"},
 	}
 	cmd.Flags().BoolVarP(&o.Follow, "follow", "f", o.Follow, "Specify if the logs should be streamed.")
+	cmd.Flags().MarkShorthandDeprecated("follow", "please use --follow instead")
```

Comment:
Looks like it is deprecated in favor of ```-F``` https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#flag-conventions.  I agree with @thockin though. ```-f``` makes more sense.